google/resource_kms_key_ring.go

```diff
@@ -0,0 +1,90 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/cloudkms/v1"
+	"log"
+)
+
+func resourceKmsKeyRing() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKmsKeyRingCreate,
+		Read:   resourceKmsKeyRingRead,
+		Delete: resourceKmsKeyRingDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"location": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func createKmsResourceParentString(project, location string) string {
+	return fmt.Sprintf("projects/%s/locations/%s", project, location)
+}
+
+func createKmsResourceKeyRingName(project, location, keyRingName string) string {
+	return fmt.Sprintf("%s/keyRings/%s", createKmsResourceParentString(project, location), keyRingName)
+}
+
+func resourceKmsKeyRingCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	name := d.Get("name").(string)
+	location := d.Get("location").(string)
+
+	parent := createKmsResourceParentString(project, location)
+
+	_, err = config.clientKms.Projects.Locations.KeyRings.
```

Comment:
style nit: for consistency with other resources, I'd recommend either not breaking at all or breaking after `Create(`

Reply:
Sorry, I'm not sure what you mean by this.  Could you give me another resource to look at as an example?